### DISCUSSION TITLE
Issues with NER with BERT_BASE in Spacy

### DISCUSSION THREAD

**Discussion by JessitaMS** [explosion/spaCy#8612] [Help: Coding & Implementations]
Hello:

I'm a new one in this of NLP. 

I'm just trying to use a pretrained model to extract the PERSON that I have in my text. First at all, I used the pretrained model **es_core_news_md**. This mark as PERSON the following words, if you can see mark as well Bucéfalo de Alejandro, Babieca, but borró is a verb, so I don't why this is marked.

![image](https://user-images.githubusercontent.com/26332548/124460769-0e4eb380-dd90-11eb-9167-2b261b975db1.png)

Then I'm trying to use **"bert-base-multilingual-cased"** with Transformer:

![image](https://user-images.githubusercontent.com/26332548/124461022-6e455a00-dd90-11eb-9319-7c2cb0320322.png)

But they don't have in the pipeline the name "NER", so I tried to add the pipe created NER, but also it don't seems work o.0

![image](https://user-images.githubusercontent.com/26332548/124461254-aea4d800-dd90-11eb-9a20-f9ffe26bbc6e.png)

Anyone can help me with this issues, cause I can't use _displacy_ because don't recognized the NER component..

Thanks!

**Comment by polm** [comment]
> This mark as PERSON the following words, if you can see mark as well Bucéfalo de Alejandro, Babieca, but borró is a verb, so I don't why this is marked.

The models make mistakes sometimes, see #3052. 

> Then I'm trying to use "bert-base-multilingual-cased" with Transformer:

I'm not sure about the way you're trying to make your pipeline, but it looks like you're using v2 style? We actually have an official Spanish Transformer model, `es_dep_news_trf`, which should be easier to use - you can just use it the same way you do the medium model. It doesn't include an NER pipeline, so you'd have to train your own, but that's also true for arbitrary BERT models. (I would also recommend against using a multilingual model if you're just working with Spanish.)

Also, please do not post screenshots of code, copy and paste the code in your question.
